source/importing-css-assets-and-javascript/index.html.md.erb

```diff
@@ -15,9 +15,37 @@ To import all the Sass rules from GOV.UK Frontend, add the following to your Sas
 @import "node_modules/govuk-frontend/govuk/all";
 ```
 
-### Import an individual component's CSS
+### Import specific parts of the CSS
 
-To import the [button](https://design-system.service.gov.uk/components/button/) component for example, add the following to your Sass file:
+If you want to improve how quickly your service's pages load in browsers, you can import only the Sass rules you need.
+ 
+1. Import `node_modules/govuk-frontend/govuk/base` in your Sass file.
+2. Import the parts of the CSS you need.
+
+For example, add the following to your Sass file to import the CSS you need for a basic GOV.UK page.
+
+```scss
+@import "node_modules/govuk-frontend/govuk/base";
+
+@import "node_modules/govuk-frontend/govuk/core/all";
+@import "node_modules/govuk-frontend/govuk/objects/all";
+@import "node_modules/govuk-frontend/govuk/components/footer/index";
+@import "node_modules/govuk-frontend/govuk/components/header/index";
+@import "node_modules/govuk-frontend/govuk/components/skip-link/index";
+@import "node_modules/govuk-frontend/govuk/utilities/all";
+@import "node_modules/govuk-frontend/govuk/overrides/all";
+```
+You can remove lines that import parts of the CSS you do not need. 
+
+[Read more about the different parts of GOV.UK Frontend’s CSS](https://github.com/alphagov/govuk-frontend/tree/master/src/govuk).
```

Comment:
Non-blocking but this should really link to https://github.com/alphagov/govuk-frontend/tree/master/src/govuk#the-src-directory